pip/req/req_uninstall.py

```diff
@@ -47,7 +47,7 @@ def _can_uninstall(self):
         return True
 
     def add(self, path):
-        path = normalize_path(path)
+        path = os.path.normcase(os.path.expanduser(path))
```

Comment:
how about a "follow_symlink" kwarg to normalize_path

add a simple unit test for `normalize_path` and `UninstallPathSet.add`.   we have no tests currently for `UninstallPathSet`

* for normalize_path:  tests/unit/test_utils.py
* for add: `tests/unit/test_req_uninstall.py`  (new module)